tests/test_modelutils.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import unittest
+
+import astropy.units as u
+import numpy as np
+
+import pint.models.model_builder as mb
+import pint.toa as toa
+import test_derivative_utils as tdu
+from pint.residuals import Residuals
+from pinttestdata import datadir
+
+from pint.modelutils import (
+    convert_to_ecliptic,
+    convert_to_equatorial,
+)
+
+
+class TestEcliptic(unittest.TestCase):
+    """Test conversion from equatorial <-> ecliptic coordinates, and compare residuals."""
+
+    @classmethod
+    def setUpClass(cls):
+        # J0613 is in equatorial
+        cls.parfileJ0613 = os.path.join(
+            datadir, "J0613-0200_NANOGrav_dfg+12_TAI_FB90.par"
+        )
+        cls.timJ0613 = os.path.join(datadir, "J0613-0200_NANOGrav_dfg+12.tim")
+        cls.toasJ0613 = toa.get_TOAs(
+            cls.timJ0613, ephem="DE405", planets=False, include_bipm=False
+        )
+        cls.modelJ0613 = mb.get_model(cls.parfileJ0613)
```

Comment:
mb.get_model() should be get_model(), if we do 
```
from pint.models import get_model
```
So does for the other ones.

Reply:
Ok, I've fixed both of these. FYI, I copied/pasted this from another test function, so it might be good to check these and be consistent. I can do that soon.